system-monitor/MonitorThread.py

```diff
@@ -19,19 +20,23 @@ def is_stopped(self):
 
     def set_pids(self, pids):
         self.pids = pids
-    
+
     def set_function_name(self, name):
         self.function_name = name
 
     def run(self):
-        top_cmd = ['top','-b','-n 1', '-p', self.pids]
+        top_cmd = ['top', '-b', '-n 1', '-p', self.pids]
         tail_cmd = ['tail', '-n+8']
         awk_cmd = ['awk', '{print $1 "\t" $9 "\t" $10}']
-        
+
         date_time = datetime.now()
         experiment_datetime = date_time.strftime("%d_%m_%Y_%H_%M_%S")
-        path = "experiments/"+ self.function_name
-        os.mkdir(path)
+        path = "./opening-black-box/experiments/" + self.function_name
+        try:
+            os.mkdir("./opening-black-box/experiments")
+            os.mkdir(path)
+        except:
+            print("Dir already exists")
```

Comment:
Why are you using part of word in "logging"? You can use full name - "Directory".

Reply:
True, i will change that